recipes/pyspark/Dockerfile

```diff
@@ -0,0 +1,12 @@
+#From should change to the dockerhub link once the pyspark ml runtime is pushed
+FROM adobe/acp-dsw-ml-runtime-pyspark:0.0.1
+RUN mkdir /recipe
+
+COPY . /recipe
+
+RUN cd /recipe && \
+    ${PYTHON} setup.py clean install && \
+    rm -rf /recipe
+
+RUN cp /databricks/conda/envs/${DEFAULT_DATABRICKS_ROOT_CONDA_ENV}/lib/python3.6/site-packages/pysparkretailapp-*.egg /application.egg
```

Comment:
I suppose we could create an ENV var called `SITE_PACKAGES` or similar in the base image and set it to the value of `/databricks/conda/envs/${DEFAULT_DATABRICKS_ROOT_CONDA_ENV}/lib/python3.6/site-packages` which would make this less obviously Databricks.  However, we can't do that until we have the base image updated.